### DISCUSSION TITLE
Sqitch affected by Docker ending Free Team organizations?

### DISCUSSION THREAD

**Discussion by ewie** [theory/sqitch#723] [Q&A]
There's news that Docker is ending Free Team organizations (https://news.ycombinator.com/item?id=35154025) and that affected organizations and their images will be deleted (https://blog.alexellis.io/docker-is-deleting-open-source-images/).

I rely on the Sqitch image in several CI pipelines and I wonder if https://hub.docker.com/u/sqitch is affected by this?

**Comment by theory** [comment]
Dug into it, and yes, apparently so. I've applied for Docker OSS status for the project, will see what they say.

**Reply by ewie** [reply]
Thanks David! I bet they get a lot of request now.

**Reply by ewie** [reply]
I assume this is no longer an issue with https://www.docker.com/blog/no-longer-sunsetting-the-free-team-plan/.

**Reply by theory** [reply]
Yeah, but I'm also applying for official OSS status.
